paths/paths.go

```diff
@@ -0,0 +1,62 @@
+package paths
```

Comment:
This probably shouldn't be a top-level package, but thanks for breaking it out. I don't have a concrete better suggestion at the moment.